src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -114,30 +116,44 @@ export const View = (props: Props) => {
   const tabSpecificExposures = getTabSpecificExposures(selectedTab);
   const filteredExposures = filterExposures(tabSpecificExposures, filters);
 
+  // MNTOR-2226
+  const getExposureCardLink = (exposure: Exposure) => {
```

Comment:
Hmm, I think we should use the `getNextGuidedStep` function here instead. Right now, we're hardcoding e.g. a link to the `high-risk-data-breaches` page, but we'll want to skip that page if the user does not have high-risk breaches.

IIRC the reason we wanted to only have a single card open at all times, is that this CTA would go to the same location for each of them, so I don't think we'd need conditionals here to make it depend on the specific exposure.

(I also think Mel had a different button label in mind because of that, but you might want to check with her on that.)

Reply:
Very cool, looked at the function, yea I think you are right, it would be fitting. I skipped the second arg for `getNextGuidedStep`, see L138